lib/include/Spix/QtQmlBot.h

```diff
@@ -9,6 +9,7 @@
 #include <QObject>
 #include <Spix/TestServer.h>
 #include <memory>
+#include "spix_export.h"
```

Comment:
Same as above :)
Could you add an empty line above this include to make clang-format happy?

```suggestion

#include "spix_export.h"
```